addons/io_scene_gltf/camera.py

```diff
@@ -2,7 +2,23 @@
 
 
 def create_camera(op, idx):
-    camera = op.gltf['cameras'][idx]
-    name = camera.get('name', 'cameras[%d]' % idx)
-    data = bpy.data.cameras.new(name)
-    return data
+    """
+    create a new blender camera from the gltf data
+    """
+    data = op.gltf['cameras'][idx]
+    name = data.get('name', 'cameras[%d]' % idx)
+    camera = bpy.data.cameras.new(name)
+    if data["type"] == "orthographic":
+        camera.type = "ORTHO"
+        p = data["orthographic"]
+        camera.clip_start = p["znear"]
+        camera.clip_end = p["zfar"]
+        camera.ortho_scale = max(p["xmag"], p["ymag"])
+    else:
```

Comment:
Can you do

    elif data["type"] == "perspective":
        ...
    else:
        raise Exception("unknown camera type: %s" % data["type"])

for a little bit more robustness instead?

Reply:
Sure, but I don't know what is best for the user; discard an unknown camera type or see a dialog with the stacktrace? Probably is better to ignore the camera (returning None?) so that the user ends with an imported scene (even if incomplete). What do you think?

Reply:
Most other places raise an exception in this case which makes it easier to find and fix. But you're right, it's better for the user if they get something at least :) I wouldn't return `None` though; maybe just print a note with the error and then return `camera`.

You can do whatever (as far as I'm concerned) as long as unknown camera types don't go down the `"perspective"` path :+1: